modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPConfigurationEntryLocalServiceImpl.java

```diff
@@ -352,7 +343,6 @@ public CPConfigurationEntry updateCPConfigurationEntry(
 		cpConfigurationEntry.setShippingExtraPrice(shippingExtraPrice);
 		cpConfigurationEntry.setShipSeparately(shipSeparately);
 		cpConfigurationEntry.setTaxExempt(taxExempt);
-		cpConfigurationEntry.setVisible(visible);
```

Comment:
is it correct that we are not removing the parameter from the sign of the method?